consensus/validator.go

```diff
@@ -356,7 +356,7 @@ func (consensus *Consensus) getPriKeysInCommittee() []*bls.PrivateKeyWrapper {
 
 func (consensus *Consensus) constructP2pMessages(msgType msg_pb.MessageType, payloadForSign []byte, priKeys []*bls.PrivateKeyWrapper) []*NetworkMessage {
 	p2pMsgs := []*NetworkMessage{}
-	if consensus.AggregateSig {
+	if consensus.AggregateSig && len(priKeys) != 0 {
```

Comment:
no need to change the logic. just help change the error message to be more explicit. instead of "No private keys provided". do "There is no elected BLS keys provided"

Reply:
Sure, OK. Done